app/templates/explore.html

```diff
@@ -6,15 +6,237 @@
+          <li><a>Intermediate</a></li>
+          <li><a>Expert</a></li>
+      </ul>
+    </div>
+
+    <div class="row">
+      <div class="col-4">
+        <div class="card slide-up">
+          <div class="card-container">
+              <div class="card-image" style="background-image: url(https://unsplash.it/640/480?random&blur)"></div>
+          </div>
+          <div class="mobile-title">
+              <div class="content">
+                  <div class="tile">
+                      <div class="tile__container">
+                          <p class="tile__title">Kangaroo Valley Safari</p>
+                          <p class="tile__subtitle">By John Doe</p>
+                      </div>
+                  </div>
+              </div>
+          </div>
+          <div class="card-body content">
+            <p>Located two hours south of Sydney in the Southern Highland of New South Wales...</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="card slide-up">
+          <div class="card-container">
+              <div class="card-image" style="background-image: url(https://unsplash.it/640/480?random&blur)"></div>
+          </div>
+          <div class="mobile-title">
+              <div class="content">
+                  <div class="tile">
+                      <div class="tile__container">
+                          <p class="tile__title">Kangaroo Valley Safari</p>
+                          <p class="tile__subtitle">By John Doe</p>
+                      </div>
+                  </div>
+              </div>
+          </div>
+          <div class="card-body content">
+              <p>Located two hours south of Sydney in the Southern Highland of New South Wales...</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="card slide-up">
+          <div class="card-container">
+              <div class="card-image" style="background-image: url(https://unsplash.it/640/480?random&blur)"></div>
+          </div>
+          <div class="mobile-title">
+              <div class="content">
+                  <div class="tile">
+                      <div class="tile__container">
+                          <p class="tile__title">Kangaroo Valley Safari</p>
+                          <p class="tile__subtitle">By John Doe</p>
+                      </div>
+                  </div>
+              </div>
+          </div>
+          <div class="card-body content">
+              <p>Located two hours south of Sydney in the Southern Highland of New South Wales...</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+  <div class="section whitebackground">
+    <div class="row">
+      <div class="col-6">
+        <h3>Collaborate with Others</h3>
+        <p>By choosing a challenge to tackle, you will have the option to work with other individuals in order to choose your favorite challenges. </p>
```

Comment:
"By choosing a challenge ... you will have the option ... to choose your challenge" this seems cyclical.

Reply:
Ditto.